core/src/main/java/io/grpc/internal/ClientTransport.java

```diff
@@ -136,12 +136,12 @@ ClientStream newStream(MethodDescriptor<?, ?> method,
 
     /**
      * Invoked when a ping is acknowledged. The given argument is the round-trip time of the ping,
-     * in microseconds.
+     * in nanoseconds.
      *
-     * @param roundTripTimeMicros the round-trip duration between the ping being sent and the
+     * @param roundTripTimeNanos the round-trip duration between the ping being sent and the
      *     acknowledgement received
      */
-    void pingAcknowledged(long roundTripTimeMicros);
+    void pingAcknowledged(long roundTripTimeNanos);
```

Comment:
Although this is internal API, there may be people using it. This would give them wrong data and it's almost impossible for them to notice it. Please keep the time as micros.